frontend/src/userContext.tsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
 import { User } from './types';
+
+const defaultFields = {
+  currentUser: {} as User,
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  setUser: (() => {}) as (user: User) => void,
```

Comment:
Here is where we can use `{} as User` to avoid getting the type error for not having the required fields